### PR TITLE
demo script

### PR DESCRIPTION
**Pull request by ragnika**
usage and output files listed in README

### CODE REVIEW

File: demo/helper.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+function base64encode(input) {
+  return Buffer.from(input).toString('base64url');
+}
+
+function base64decode(input) {
+  return Buffer.from(input, 'base64url').toString();
+}
+
+const hasher = (data) => {
```
Comment (linasi) [inline demo/helper.js:27]: @ragnika any reason for this to be defined as an anonymous function assigned to the variable and all above by using `function` keyword?

File: demo/issue.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    const disclosure = base64decode(d);
+    const arr = JSON.parse(disclosure);
+
+    if (arr.length == 2) {
+      file.write('__Array Entry__:  \n\n');
+    } else {
+      file.write('__Claim `'+arr[1]+'`__:help  \n\n');
+    }
+
+    const hash = hasher(d);
```
Comment (linasi) [inline demo/issue.js:18]: @ragnika minor thing, but I would put a new line after this statement

**Review by linasi — APPROVED** [review]
👍

File: demo/helper.js (hunk 1 above)
Comment (ragnika) [reply demo/helper.js:27]: @linasi, nah, i just didn't pay enough attention when i was moving stuff around and the scripts into this repo. Thanks for pointing it out

**Pull request merged**